src/ParameterInformation.php

```diff
@@ -9,10 +9,16 @@
 class ParameterInformation
 {
     /**
-     * The label of this signature. Will be shown in
-     * the UI.
+     * The label of this parameter information.
      *
-     * @var string
+     * Either a string or an inclusive start and exclusive end offsets within its containing
+     * signature label. (see SignatureInformation.label). The offsets are based on a UTF-16
+     * string representation as `Position` and `Range` does.
+     *
+     * *Note*: a label of type string should be a substring of its containing signature label.
+     * Its intended use case is to highlight the parameter label part in the `SignatureInformation.label`.
+     *
+     * @var string|array
```

Comment:
```suggestion
     * @var string|int[]
```

Reply:
Of course, not sure why that didn't cross my mind 😄